### PR TITLE
docs: ternary relationships

### PR DESCRIPTION
**Pull request by rafaelkallis**
Added Ternary relationships to docs.

### CODE REVIEW

File: doc/includes/RECIPES.md [hunk 1]
```diff
@@ -778,3 +778,232 @@ explain select * from "Hero" where details#>>'{type}' = 'Hero';
...
+// User.js
+const { Model } = require("objection");
+
+class User extends Model {
+  static get tableName() { return "user"; }
+  static get relationMappings() {
+    return {
+      groups: {
+        relation: Model.ManyToManyRelation,
+        modelClass: require("Group"),
```
Comment (koskimas) [inline doc/includes/RECIPES.md]: This should be require('./Group') or some other relative path. Not super important for this example, but it's better to be precise

Comment (koskimas) [inline doc/includes/RECIPES.md]: Same goes for other relation mappings..

File: doc/includes/RECIPES.md [hunk 2]
```diff
@@ -778,3 +778,232 @@ explain select * from "Hero" where details#>>'{type}' = 'Hero';
...
+UserGroupPermission
+  .query()
+  .select(
+    "first_name",
+    "last_name",
+    "label",
+    "extra_attribute"
+  )
+  .joinRelation("user")
+  .joinRelation("permission")
```
Comment (koskimas) [inline doc/includes/RECIPES.md]: joinRelation now takes relation expressions. These two calls can be replaced with `joinRelation('[user, permission]')`

**Comment by rafaelkallis** [comment]
Thanks for the feedback, updated the PR.

**Pull request merged**